lib/manager/gitlabci/extract.ts

```diff
@@ -120,15 +125,15 @@ export async function extractAllPackageFiles(
     if (is.array(doc?.include)) {
       for (const includeObj of doc.include) {
         if (is.string(includeObj.local)) {
-          const fileObj = includeObj.local.replace(/^\//, ''); // TODO #12071 #12070
+          const fileObj = trimLeadingSlash(includeObj.local); // TODO #12071
```

Comment:
```suggestion
          const fileObj = trimLeadingSlash(includeObj.local);
```

I don't think that issue is applicable now